yos/sort_list.go

```diff
@@ -0,0 +1,43 @@
+package yos
+
+type SortListByName []*FilePathInfo
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type SortListByName should have comment or be unexported](https://app.codacy.com/manual/an9an63/gut/pullRequest?prid=4963210)